lightapi/config.py

```
@@ -56,6 +56,9 @@ def jwt_algorithm(self) -> str:
 class Authentication:
     """Authentication configuration for a RestEndpoint."""
 
+    # Standard JWT reserved claims that cannot be used as extra claims
+    RESERVED_CLAIMS = {"exp", "iat", "nbf", "iss", "sub", "aud", "jti"}
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)